homeassistant/components/logger.py

```diff
@@ -23,6 +27,22 @@
 LOGGER_DEFAULT = 'default'
 LOGGER_LOGS = 'logs'
 
+_LOGS_SCHEMA = vol.All(
+    cv.ensure_list,
```

Comment:
This should not be a list, just the `vol.Schema({…})` part is needed.

From the code line 84:

```python
for key, value in config.get(DOMAIN)[LOGGER_LOGS].items():
    …
```